src/index.js

```diff
@@ -32,13 +52,22 @@ function sendNotification(params, settings) {
     body: JSON.stringify({
       text: `KAHOLO NOTIFICTION:
         Pipeline: ${pipeline.name} (${pipeline.id})
-        Start Time: ${formatDate(startTime)}
-        Finish Time: ${formatDate(finishTime)}
-        Execution Id: ${executionId},
-        type: ${type},
-        message: ${message},
-        process: ${process?.name} (${process?.id})
-        action: ${action?.name} (${action?.id})
+        Agent: ${pipeline.agentName}
```

Comment:
I would add pipeline status to the message. It might be useful for our testing/debugging purposes.